libs/domain/picking/src/components/picking-list-item/stories/demo.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { Meta, Story } from '@storybook/web-components';
+import { html } from 'lit';
+import { TemplateResult } from 'lit/development';
+import { storybookPrefix } from '../../../../.constants';
+import { mockPickingListData } from '../../../mocks';
+
+const pickingListIds = mockPickingListData.map(({ id }) => id);
+
+export default {
+  title: `${storybookPrefix}/Picking list item`,
+  args: {
+    pickingListId: pickingListIds[0],
+  },
+  argTypes: {
+    pickingListId: {
+      options: pickingListIds,
+      control: { type: 'select' },
+    },
+  },
+} as Meta;
+
+interface Props {
+  pickingListId: string;
```

Comment:
Create an interface for `PickingListItemComponent` please, and reuse it here as a type for props.